### PR TITLE
31424 update card style to match cms list design

### PR DESCRIPTION
**Pull request by AlejandroColinaGSA**

### CODE REVIEW

**Comment by AlejandroColinaGSA** [comment]
Annotation 2020-03-04 172941

File: src/stylesheets/components/_card.scss [hunk 1]
```diff
@@ -1,29 +1,43 @@
...
-  @include u-padding-x('205');
-  @include u-padding-bottom('205');
+  @include u-margin-y(2);
+  // @include u-padding-top(2);
+  // @include u-padding-x("205");
+  // @include u-padding-bottom("205");
   border-radius: 4px;
   background-color: white;
-  box-shadow: 0 10px 25px 5px rgba(0, 0, 0, 0.15);
+  box-shadow: 0 6px 12px 0px rgba(0, 0, 0, 0.1);
```
Comment (davereed) [inline src/stylesheets/components/_card.scss]: Let's use a value from the USWDS library if possible, work with the designer to adjust if needed.

https://designsystem.digital.gov/utilities/shadow/

File: src/stylesheets/components/_card.scss [hunk 2]
```diff
@@ -36,10 +50,49 @@
...
+  @include u-padding-bottom("205");
   display: block;
-  @include u-padding-top('105');
+  @include u-padding-top("105");
+  @media only screen and (max-width: 600px) {
+    display: none;
+  }
+}
+.sds-card--vertical .sds-card__content {
+  padding-left: 16px;
```
Comment (davereed) [inline src/stylesheets/components/_card.scss]: Use utils - u-padding-left(2) ?

File: src/stylesheets/components/_card.scss [hunk 3]
```diff
@@ -36,10 +50,49 @@
...
+  @include u-padding-top(2);
+}
+.sds-card--title {
+  padding-top: 0;
+  margin-top: 0;
```
Comment (davereed) [inline src/stylesheets/components/_card.scss]: util imports?

File: src/stylesheets/components/_card.scss [hunk 4]
```diff
@@ -36,10 +50,49 @@
...
+  }
+}
+.sds-card--vertical .sds-card__header-mobile {
+  margin-right: auto;
+  @include u-margin-left("05");
+  @include u-padding("205");
+  @media only screen and (min-width: 600px) {
     display: none;
+    overflow: hidden;
+    max-height: 0;
```
Comment (davereed) [inline src/stylesheets/components/_card.scss]: Use USWDS utils:
https://designsystem.digital.gov/utilities/height-and-width/

File: src/stylesheets/components/_card.scss [hunk 5]
```diff
@@ -36,10 +50,49 @@
...
+    @include u-padding-bottom("205");
+    @include u-border-right(0);
+  }
+}
+.sds-card--vertical .sds-card__header-mobile {
+  margin-right: auto;
+  @include u-margin-left("05");
+  @include u-padding("205");
+  @media only screen and (min-width: 600px) {
     display: none;
```
Comment (davereed) [inline src/stylesheets/components/_card.scss]: Convert to util method

File: src/stylesheets/components/_card.scss [hunk 6]
```diff
@@ -36,10 +50,49 @@
...
+  padding-left: 16px;
+  @include u-padding-top(2);
+}
+.sds-card--title {
+  padding-top: 0;
+  margin-top: 0;
+}
+
+.sds-card--subtitle {
+  // margin-top: -1.3rem;
```
Comment (davereed) [inline src/stylesheets/components/_card.scss]: remove commented code

File: src/stylesheets/components/_card.scss [hunk 7]
```diff
@@ -1,29 +1,43 @@
 .sds-card {
-  @include u-margin-y(5);
-  @include u-padding-top(2);
-  @include u-padding-x('205');
-  @include u-padding-bottom('205');
+  @include u-margin-y(2);
+  // @include u-padding-top(2);
+  // @include u-padding-x("205");
+  // @include u-padding-bottom("205");
```
Comment (davereed) [inline src/stylesheets/components/_card.scss]: remove commented code

**Pull request merged**